env-files/torch/jupyter/configure.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# Derived from (restricted access):
+# https://gitlab.cern.ch/escape-wp2/docker-images/-/blob/master/datalake-singleuser/bin/configure.py
+
+import configparser
+import json
+import os
+
+# HOME = '/home/jovyan'
+# HOME = '/ceph/hpc/home/ciangottinid'
+
+
+def write_jupyterlab_config():
+    HOME = os.getenv('HOME', '/ceph/hpc/home/ciangottinid')
+
+    file_path = HOME + '/.jupyter/jupyter_notebook_config.json'
+    if not os.path.isfile(file_path):
+        os.makedirs(HOME + '/.jupyter/', exist_ok=True)
```

Comment:
This should work, but I would use pathlib here for best practice

Reply:
that's just a file I copied from another repo actually, which I am not maintaining